src/main/java/frc/robot/subsystems/elevator/C2025ElevatorIO.java

```diff
@@ -0,0 +1,99 @@
+package frc.robot.subsystems.elevator;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.controls.MotionMagicVoltage;
+import com.ctre.phoenix6.controls.NeutralOut;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.system.plant.LinearSystemId;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.simulation.DCMotorSim;
+import frc.robot.Robot;
+
+public class C2025ElevatorIO implements ElevatorIO {
+    private final TalonFX motor;
+    private final TalonFX motor2;
+
+    private final MotionMagicVoltage positionRequest = new MotionMagicVoltage(0.0);
+    private final NeutralOut stopRequest = new NeutralOut();
+
+    private static final double GEAR_RATIO = 1.0;
+    private static final double PULLEY_RADIUS = Units.inchesToMeters(1.0);
+    private static final double PULLEY_CIRCUMFERENCE = (2 * Math.PI * PULLEY_RADIUS);
```

Comment:
These constants should be above the `TalonFX motor` field. We should try to order our fields as follows: `public static`, `private static` `public` `private`.